packages/stories/src/index.js

```diff
@@ -1,5 +1,2 @@
-export { storiesOf } from '@storybook/react';
-export * from '@storybook/addon-knobs';
-export * from '@storybook/addon-actions';
-export * from '@storybook/addon-storysource';
```

Comment:
Storybook didn't render when both of these were exported.  Probably not safe to `export * from` multiple npm packages when some of the exports could have the same names.